hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -708,8 +696,8 @@ boolean canSizeEnterTarget(DatanodeDetails target, long size) {
       //2 current usage of target datanode plus sizeEnteringAfterMove
       // is smaller than or equal to upperLimit
       return sizeEnteringAfterMove <= config.getMaxSizeEnteringTarget() &&
-           nodeManager.getUsageInfo(target)
-               .calculateUtilization(sizeEnteringAfterMove) <= upperLimit;
+           Double.compare(nodeManager.getUsageInfo(target)
+               .calculateUtilization(sizeEnteringAfterMove), upperLimit) < 1;
```

Comment:
```suggestion
               .calculateUtilization(sizeEnteringAfterMove), upperLimit) <= 0;
```

Reply:
the return type of `Double.compare` is int , so `< 1`  is equal to `<= 0` here. but `<=0` seems more clear, will fix this, thanks.